File: C_Programming/Competitive/program26.c
//////  Write a program which accepts a number from user and if the number is less than 50, then print small, if it is greater than 50, and less than 100,
////// then print medium and if it is greater than 100, then print large.

//////  Input : 75
//////  Output : Medium

#include<stdio.h>

void Number(int iNo)
{
    if (iNo < 50)
    {
        printf("Small");
    }
    else if (iNo > 50 && iNo < 100)
    {
        printf("Medium");
    }
    else 
    {
        printf("Large");
    }
}

int main()
{
    int iValue = 0;

    printf("Enter a number : \n");
    scanf("%d", &iValue);

    Number(iValue);
    
    return 0;
}

File: Java_Programming/Competitive/program26.java
////    Write a program which accepts a number form user and make the 7th bit OFF of that number if it is ON.
///     Return modified number.
///     Input : 79;
///     Output : 15

import java.util.*;

class program26
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        int iRet = 0;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();

        iRet = bobj.OffBit(iValue);

        System.out.println("Modified number is : " + iRet);

        sobj.close();
    }
}

class Bitwise
{
    int OffBit(int iNo)
    {
        int iMask = 0x40;
        
        return (iNo ^ iMask);
    }
}